spec/features/resource_manager_manages_holidays_spec.rb

```diff
@@ -165,28 +240,48 @@ def and_there_are_guiders_with_holidays
     end
   end
 
-  def and_they_edit_the_holiday
+  def and_they_edit_the_multiple_day_holiday
```

Comment:
This step made me weep solemnly. The `start_at` `end_at` stuff is appearing elsewhere and a bunch of the other stuff too. We should try and push as much of this into the page as possible. The steps should be as imperative as we can make them.